discovery_engine/lib/src/domain/search_manager.dart

```diff
@@ -185,6 +186,21 @@ class SearchManager {
     return EngineEvent.searchTermRequestSucceeded(search.searchTerm);
   }
 
+  /// Return the current trending topics.
+  Future<EngineEvent> trendingTopicsRequested() async {
+    final topics = await _engine.getTrendingTopics();
+
+    // TODO: do we need to persist the engine state??
```

Comment:
this is difficult to answer right now, because this is not yet implemented in the rust DE, so it might change later on.

Reply:
https://xainag.atlassian.net/browse/TY-2807